README.md

```diff
@@ -53,3 +53,11 @@ Base HD Path:  m/44'/60'/0'/0/{account_index}
 2019-07-15 08:18:55,488 INFO  [ws] Listening for new connections on 127.0.0.1:8546.
 2019-07-15 08:18:55,492 INFO  [oasis_chain] Oasis local chain is running
 ```
+
+## Docker
+
+You can also run it as a Docker container:
+
+```
+$ docker run --rm -p 127.0.0.1:8546:8546 oasislabs/oasis-chain:latest
```

Comment:
```suggestion
docker run --rm -p 127.0.0.1:8546:8546 oasislabs/oasis-chain:latest
```

Reply:
`$` makes copy-pasting that much harder

Reply:
It is used above in command example. Also, I like it more because if you just copy-paste nothing happens. Which I think is good. And if you really want just to copy-paste and not think, you have to select a bit of it. So i find this a perfect balance between warning "do not copy-paste blindly" and "OK, but be careful".

Reply:
it doesn't generalize, though. say you added another line and now you definitely can't copy+paste it.

Reply:
Yes, you should copy paste one by one anyway, no?

They can still copy-paste, just be careful about it.

Reply:
> you should copy paste one by one anyway

mmm I've gotta disagree with you there. I'd prefer to just skim it, copy the whole block, and paste it in--especially if I've already reviewed the code block once and just want to repeat the process.

Reply:
So we reached an impasse here.

So, should we go by the fact that other examples in the same document use `$`?

Reply:
only because you got to merge them before this one. hrmph!

Reply:
That was not me: https://github.com/oasislabs/oasis-chain/commit/37ea425b5b8ddaad6d637e00bb1ba4946da7f69e

Reply:
But @peterjgilbert.